src/app/core-ui/main/release-notification/release-notification.component.ts

```diff
@@ -16,23 +16,23 @@ export class ReleaseNotificationComponent implements OnInit, OnDestroy {
   public currentClientVersion: string = environment.version;
   public latestClientVersion: string;
   public releaseUrl: string;
-  private destroyed: boolean = false;
+  private destroyed: boolean = true;
 
   constructor(private clientVersionService: ClientVersionService) { }
 
   ngOnInit() {
-    this.getCurrentClientVersion();
+    // check new update in every 30 minute
+    const versionInterval = interval(1800000);
+    versionInterval.takeWhile(() => !this.destroyed).subscribe(val => this.getCurrentClientVersion());
   }
 
+  // no need to destroy.
```

Comment:
there is a need to destroy...

Reply:
I can see that component is never been destroyed !!!

Reply:
Maybe not in the current state, but in the future it will.

Reply:
Okay, so I'll remove that comment line :+1: